src/new/connection.js

```diff
@@ -97,6 +98,18 @@ class Connection extends EventEmitter {
     this.websocket.on('is_online', this.onConnectionChange);
     this.websocket.on('wallet', this.handleWalletMessage);
 
+    this.websocket.on('reload_data', () => {
+      this.emit('reload-data');
+    });
+
+    this.websocket.on('height_updated', (height) => {
+      this.emit('height-updated', height);
```

Comment:
I'd rename the event to `best-block-update`.

Reply:
Makes sense, done.